scripts/manageFWrobot.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
```

Comment:
```suggestion
#!/usr/bin/env python3
```

By using `/usr/bin/env` and not hardcoding `/usr/bin/python3`, this will work fine in virtual environments.